utils/fake_data.py

```diff
@@ -41,6 +51,44 @@ def iso_date_time(self):
         """
         return datetime.isoformat(self.date_time())
 
+    def set_safe_name(self):
+        """
+        Randomly selects a safe full name to use.
+        """
+        random_name = self.random_element(self.safe_data["names"])
+        self.first_name_used, self.last_name_used = False, False
+
+        self.current_name.update({"first_name": random_name["first_name"], "last_name": random_name["last_name"]})
+
+    def safe_first_name(self):
+        """
+        Returns a safe first name as a string.
+        """
+        if not self.first_name_used:
+            return self.current_name["first_name"]
+        else:
+            self.set_safe_name()
+            self.first_name_used = True
+            return self.current_name["first_name"]
```

Comment:
I would refactor this method a little to avoid duplicate returns. Like so:

```python
if self.first_name_used:
    self.set_safe_name()

self.first_name_used = True
return self.current_name["first_name"]
```

And I would do the same for `safe_last_name`. And I think maybe setting the attributes `self.first_name_used` and `self.last_name_used` to True needs to happen regardless of which part of the `if` statement we're in? Although it looks like it's working this way too, but I'm not sure why 🤔